.github/ISSUE_TEMPLATE/alpha_bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: 🧪 Alpha SDK bug report
+about: Have you found a bug in one of our pre-release or alpha SDKs? File it here.
+title: ""
+labels: "alpha"
+assignees: ""
+---
+
+<!-- Step 1 [READ THIS] -->
+<!--
+Are you in the right place?
+  * For issues or feature requests related to __the code in this repository__
+    file a Github issue.
+  * If you are not yet participating in our Alpha and would like to join, use this form:
+    https://forms.gle/bKBEm48W8EMPLaUp9
+  * For general Alpha discussion, use the google group:
+    https://groups.google.com/g/firebase-js-sdk-testers
+-->
+
+<!-- Step 2 -->
+
+### [REQUIRED] Describe your environment
+
+  * Operating System version: _____
+  * Browser version: _____
+  * Firebase SDK version: _____
+  * Firebase Product: _____ (auth, database, storage, etc)
+
+<!-- Step 3 -->
+
+### [REQUIRED] Describe the problem
+
+#### Steps to reproduce:
+<!--
+  What happened? How can we make the problem occur?
+  This could be a description, log/console output, etc.
+-->
+#### Relevant Code:
+
+<!--
+  Reproduce the issue on StackBlitz and provide your forked URL
+  or give us some sample code below
+-->
+https://stackblitz.com/fork/firebase-issue-sandbox
```

Comment:
The stackblitz link isn't going to work for exp and I guess nobody even uses it for regular bug reports, which is understandable since the project template seems to be set on firebase 4.9.0. Maybe we should just remove it from both for now?

Reply:
Hah ok will do!